contract/r/gnoswap/access/access_test.gno

```diff
@@ -500,6 +500,84 @@ func TestAccess_MustGetAddress(t *testing.T) {
 	}
 }
 
+func TestSetRoleAddress_EdgeCases(t *testing.T) {
+	// Save original state
+	originalRoles := make(map[string]address)
+	for k, v := range roleAddresses {
+		originalRoles[k] = v
+	}
+	defer func() {
+		roleAddresses = originalRoles
+	}()
+
+	testAddr := testutils.TestAddress("test")
+
+	tests := []struct {
+		name        string
+		setupRoles  map[string]address
+		roleName    string
+		roleAddress address
+		shouldPanic bool
+		panicMsg    string
```

Comment:
the panic verification is never executed because none of the test cases expect a panic. It would be better to either remove the `shouldPanic`/`panicMsg` fields from the `tests`, or add test cases that cover the panic path.